src/store/app/app.selectors.ts

```diff
@@ -93,6 +93,15 @@ export const chainsInCurrentNetworkSelector = createSelector(
   }
 );
 
+export const isOnTestnetSelector = createSelector(
```

Comment:
now, that you have this selector you could use it inside the `chainsInCurrentNetworkSelector`, instead of `chainIdSelector`.